### PR TITLE
Update Rancher integration tests

### PR DESCRIPTION
**Pull request by khushboo-rancher**
Ref : https://github.com/harvester/tests/issues/1301

### CODE REVIEW

File: docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Please refer to [02-Integrate to Rancher from Harvester settings](https://harves
 3. Click import existing
 4. Copy the curl command 
 ![image.png](https://images.zenhubusercontent.com/61519853321ea20d65443929/08e70d37-e573-47b1-a3d6-0f3615116d48)
-1. SSH to harvester master node (user: rancher)
+1. SSH to harvester master node
 1. Execute the curl command to import harvester to rancher
 `curl --insecure -sfL https://192.168.50.82/v3/import/{identifier}.yaml | kubectl apply -f -`
```
Comment (noahgildersleeve) [inline docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md:14]: This isn't how we import Harvester clusters anymore. It should be
1. Create a new Harvester cluster from Virtualization Management
1. Change the `cluster-registration-url` to match the one in Rancher

File: docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ Please refer to [02-Integrate to Rancher from Harvester settings](https://harves
     deployment.apps/cattle-cluster-agent created
     service/cattle-cluster-agent created
     ```
-1. Check import status in Virtualization Management page on Rancher 
+1. Check import status in Virtualization Management page on Rancher.
```
Comment (noahgildersleeve) [inline docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md:26]: The import status will be shown as `Active` once it's online

**Review by noahgildersleeve — CHANGES_REQUESTED** [review]
There's just a couple of small things in the really old tests that should be changed. Other than that LGTM.

File: docs/content/manual/harvester-rancher/03-manage-vm-downstream-harvester.md [hunk 3]
```diff
@@ -21,3 +22,4 @@ Prerequisite: Harvester already imported to Rancher Dashboard
 1. Can `Stop`, `Restart` and `Migrate` virtual machine correctly
 1. Can `Clone` virtual machine correctly
 1. Can `Delete` virtual machine correctly
+1. Vm should be successfully created.
```
Comment (albinsun) [inline docs/content/manual/harvester-rancher/03-manage-vm-downstream-harvester.md:25]: I don't get this.
Is this the same to item 1~3?

> 1. Can create a single instance vm correctly
> 2. Can create multiple instances vm correctly
> 3. Can diaply all virtual machine information
> ...

File: docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md (hunk 1 above)
Comment (khushboo-rancher) [reply docs/content/manual/harvester-rancher/01-import-existing-harvester-in-rancher.md]: Thanks @noahgildersleeve @albinsun I removed the file.

File: docs/content/manual/harvester-rancher/03-manage-vm-downstream-harvester.md (hunk 3 above)
Comment (khushboo-rancher) [reply docs/content/manual/harvester-rancher/03-manage-vm-downstream-harvester.md]: Exactly, why do have this at last check :P

**Pull request merged**